app/controllers/tag_migrations_controller.rb

```diff
@@ -50,7 +50,7 @@ def destroy
   end
 
   def publish_tags
-    TagImporter::PublishTags.new(tag_migration, user: current_user).run
+    BulkTagging::PublishTags.new(tag_migration, user: current_user).run
```

Comment:
Not a huge thing, but what do you reckon about _not_ namespacing this at all until we sort out the naming fiasco?